SUPPORT.md

```diff
@@ -0,0 +1,21 @@
+# Support
+
+Thank you for your interest in this project! If you are experiencing problems
+or have questions, the following are some suggestions for how to get help.
+
+> [!NOTE]
+> Before participating in our community, please read our [code of
+> conduct](CODE_OF_CONDUCT.md). By interacting with this repository,
+> organization, or community, you agree to abide by its terms.
+
+## Report an issue or request a feature
+
+To report an issue or request a feature, please first search the [issue tracker
+on GitHub](https://github.com/quantumlib/OpenFermion/issues) to check if there
```

Comment:
change to qsim

Reply:
Oh, good grief. Thanks for catching that.